common/tools/dev-tool/src/util/resolveProject.ts

```diff
@@ -71,7 +71,7 @@ export interface ProjectInfo {
 async function isAzureSDKPackage(fileName: string): Promise<boolean> {
   const f = await import(fileName);
 
-  if ((f.name as string).startsWith("@azure/")) {
+  if ((f.name as string).startsWith("@azure/") || (f.name as string).startsWith("@azure-rest/")) {
```

Comment:
Could use a RegExp here and make it work with any namespaces we might introduce in the future as well.

```suggestion
  if (/^@azure(-[a-z]+)?\//.matches(f.name)) {
```